hera_mc/tests/__init__.py

```diff
@@ -37,3 +40,53 @@ def setUp(self):
     def tearDown(self):
         self.test_trans.rollback()
         self.test_conn.close()
+
+
+def get_iterable(x):
+    """Helper function to ensure iterability."""
+    if isinstance(x, collections.Iterable):
+        return x
+    else:
+        return (x,)
+
+
+# Functions that are useful for testing:
+def clearWarnings():
+    """Quick code to make warnings reproducible."""
+    for name, mod in list(sys.modules.items()):
+        try:
+            reg = getattr(mod, "__warningregistry__", None)
+        except ImportError:
+            continue
+        if reg:
+            reg.clear()
+
+
+def checkWarnings(func, func_args=[], func_kwargs={},
+                  category=UserWarning,
+                  nwarnings=1, message=None, known_warning=None):
+    """Function to check expected warnings."""
+    category = get_iterable(category)
+    message = get_iterable(message)
```

Comment:
Lets use the updates that I just did on pyuvdata here, so that category and message get expanded if needed.